utils/clflags.ml

```diff
@@ -370,6 +370,7 @@ let dump_into_file = ref false (* -dump-into-file *)
 
 type 'a env_reader = {
   parse : string -> 'a option;
+  print : 'a -> string;
   usage : string;
   env_var : string;
```

Comment:
We would have the option of adding `default` here, although I think that the different way you did it is fine.

Reply:
Hum good point, although I'll stay with the current version I think.